internal/provider/resource/project_role_resource.go

```diff
@@ -301,13 +301,21 @@ func (r *projectRoleResource) Read(ctx context.Context, req resource.ReadRequest
 			}
 		}
 
-		permissionPlan = append(permissionPlan, projectRoleResourcePermissions{
-			Action:  types.StringValue(action),
-			Subject: types.StringValue(subject),
-			Conditions: &projectRoleResourcePermissionCondition{
+		conditions := &projectRoleResourcePermissionCondition{}
+
+		if el["conditions"] == nil {
+			conditions = nil
+		} else {
+			conditions = &projectRoleResourcePermissionCondition{
 				Environment: types.StringValue(environment),
 				SecretPath:  types.StringValue(secretPath),
-			},
+			}
+		}
+
+		permissionPlan = append(permissionPlan, projectRoleResourcePermissions{
+			Action:     types.StringValue(action),
```

Comment:
I think the action is now an array according to our response schema. Can you double check?

Reply:
Already accounted for in a previous step in the code, see line 248-250